website/content/docs/job-specification/consul.mdx

```diff
@@ -76,11 +76,16 @@ to Using Workload Identity with Consul</a>
 
 The Nomad client will make the Consul token available to the task by writing it
 to the secret directory at `secrets/consul_token` and by injecting a
-`CONSUL_TOKEN` environment variable in the task. If the Nomad cluster is
-[configured][config_consul_namespace] to use [Consul Namespaces][], a
+`CONSUL_TOKEN` environment variable in the task. 
+
+If the Nomad cluster is [configured][config_consul_namespace] to use [Consul Namespaces][], a
 `CONSUL_NAMESPACE` environment variable will be injected whenever `CONSUL_TOKEN`
 is set.
 
+If the Nomad cluster is [configured][config_consul_namespace] to use [Consul Admin Partitions][], a
+`CONSUL_PARTITION` environment variable will be injected whenever `CONSUL_TOKEN`
+is set.
```

Comment:
Not sure why we're making this change, as it's not correct. We don't inject the env var for partitions.